ss_admin_console/main/tests/classes/BaseDataFeederTest.cls

```diff
@@ -0,0 +1,136 @@
+    }
+
+    @IsTest 
+    private static void testFeedUpsertMocksPositive() {
+        fflib_ApexMocks mocks = new fflib_ApexMocks();
+        FeedLogger logger = (FeedLogger) mocks.mock(FeedLogger.class);
+        BaseDataFeeder df = baseDataFeederInstance(mocks, false, true);
+        Test.startTest();
+        df.feed(true, logger);
+        Test.stopTest();
+        ((FeedLogger) mocks.verify(logger, 1)).log('Contact', '2 records were proceed.', SSACConstants.UPSERTED_STATUS);
+    }
+
+    @IsTest 
+    private static void testFeedInsertMocksNegative() {
+        fflib_ApexMocks mocks = new fflib_ApexMocks();
+        FeedLogger logger = (FeedLogger) mocks.mock(FeedLogger.class);
+        BaseDataFeeder df = baseDataFeederInstance(mocks, true, true);
+        Test.startTest();
+        df.feed(false, logger);
+        Test.stopTest();
+        ((FeedLogger) mocks.verify(logger, 1)).error(fflib_Match.stringStartsWith('FAILED'));
+    }
+
+    @IsTest 
+    private static void testFeedInsertMocksPositive() {
+        fflib_ApexMocks mocks = new fflib_ApexMocks();
+        FeedLogger logger = (FeedLogger) mocks.mock(FeedLogger.class);
+        BaseDataFeeder df = baseDataFeederInstance(mocks, false, true);
+        Test.startTest();
+        df.feed(false, logger);
+        Test.stopTest();
+        ((FeedLogger) mocks.verify(logger, 1)).log('Contact', '1 records were proceed.', SSACConstants.INSERTED_STATUS);
+    }
+
+    @IsTest 
+    private static void testFeedInsertMocksPositiveAccounts() {
+        fflib_ApexMocks mocks = new fflib_ApexMocks();
+        FeedLogger logger = (FeedLogger) mocks.mock(FeedLogger.class);
+        BaseDataFeeder df = baseDataFeederInstance(mocks, false, false);
+        Test.startTest();
+        df.feed(false, logger);
+        Test.stopTest();
+        ((FeedLogger) mocks.verify(logger, 1)).log('Account', '1 records were proceed.', SSACConstants.INSERTED_STATUS);
+    }
+    
+    private static BaseDataFeeder baseDataFeederInstance(fflib_ApexMocks mocks, Boolean clearResults, Boolean isChild) {
+        DataFilter filter = (DataFilter) mocks.mock(DataFilter.class);
+        mocks.startStubbing();
+        if (clearResults) {
+            if (isChild) {
+                resultsContact.clear();
+            } else {
+                resultsAccount.clear();
+            }
+        }
+        mocks.when(filter.filter()).thenReturn(isChild ? resultsContact : resultsAccount);
+        mocks.stopStubbing();
+        DataSupplier accountDataSupplier = new StaticDataSupplier(dataDefinitionBuilderAccount.build(), sampleTestDataAccount);
+        DataSupplier contactDataSupplier = new StaticDataSupplier(dataDefinitionBuilderContact.build(), sampleTestDataContact);
+        BaseDataFeeder df = new BaseDataFeeder(isChild ? contactDataSupplier : accountDataSupplier);
+        df.mockedDF = filter;
+        return df;
+    }
+
+    private static List<SSACResult> resultsContact = new List<SSACResult> {
```

Comment:
I'd recommend using static methods that instantiate complex data instead of static fields.
Class methods are executed only when they are called, static fields are executed every time that class is referenced. In our case, every time a test method is executed, regardless if the field is in used or not.